src/components/form/examples/form-layout.tsx

```diff
@@ -4,6 +4,49 @@ import { schema } from './schema-layout';
 
 /**
  * Layout
+ * By default, each item in a limel-form will be rendered in a single row; and
+ * each row occupies the entire available width of the form's container.
+ *
+ * This default layout may work fine on small screens or narrow containers,
+ * but on larger screens it won't usually produce a nice layout. Thus we
+ * recommend that you choose an appropriate responsive layout for your form.
```

Comment:
Oh good catch. 😂 This means when they don't specify `layout: grid`. Then it will be single rows which are 100% wide (1 column). 

But if the specify `layout: grid`, then we default on 5 columns. We can for sure improve how we explain it here.

Reply:
It might be that I was just confused by the commit called `feat(object-field): give limel-form a css-grid layout by default`. But I think that refers to the css-property `grid`, which isn't the same thing as what we mean by `layout: grid` here, right?

I'll read it again with those glasses on, and see if it makes sense then 👍

Reply:
Totally right. Both of them have `display: grid;` in their CSS. (because we wanted to use some advantages of grid, like the `grid-gap` even for the one columned layout.